src/pages/Pools/Home/ClosurePrompts.tsx

```diff
@@ -43,11 +44,11 @@ export const ClosurePrompts = () => {
     memberCounter === '1';
 
   // depositor needs to unbond funds
-  const depositorCanUnbond = active.toNumber() > 0 && !targets.length;
+  const depositorCanUnbond = !active.gt(new BN('0')) && !targets.length;
```

Comment:
This is not equivalent to, right?
```suggestion
  const depositorCanUnbond = active.isZero() && !targets.length;
```

Reply:
I tested it, the `1` is a mistake. I probably tried to do the `!active.isZero()` and later second guessed myself if `> 0` was intentional to safeguard against negative numbers, but I guess it shouldn't be possible here.

Reply:
It might've been that I wanted to do `!active.isZero()`, but wasn't sure if the number could be negative. And the `!` was left behind 🤔